src/core/components/auth/auths.jsx

```diff
@@ -17,6 +17,17 @@ export default class Auths extends React.Component {
     this.state = {}
   }
 
+  componentDidMount() {
+    var inputs = document.getElementsByClassName("modal-ux-content")[0].getElementsByTagName("input")
```

Comment:
As a vanilla JS solution, this looks good to me. However, thinking about this more, we really should "React-ify" this to not directly change the DOM with the `.focus`. E.g. use "refs"

I'm thinking would could try adding a ref to `className="auth-container"` [here](https://github.com/swagger-api/swagger-ui/pull/7712/files#diff-e45dd3fb35ca515a5577879c3ac47d11ef521b2a8b00eadca99cecfb2baffd8dR85) that would contain multiple current refs. Most likely then also need to add creation of refs within the two maps that generate React element. e.g. something similar to `ref={el => inputRef.current[x] = el} `.


More info on refs [here](https://reactjs.org/docs/refs-and-the-dom.html), and we'll want to be compatible with the possible future use of React hooks.